decidim-meetings/spec/commands/join_meeting_spec.rb

```diff
@@ -191,5 +193,49 @@ module Decidim::Meetings
         expect { subject.call }.to broadcast(:invalid)
       end
     end
+
+    context "when there are a registration form" do
+      let!(:questionnaire) { create(:questionnaire) }
+      let!(:question) { create(:questionnaire_question, questionnaire: questionnaire, position: 0) }
+      let(:meeting) { create :meeting, component: component, registrations_enabled: registrations_enabled, available_slots: available_slots, questionnaire: questionnaire }
+
+      let(:valid) { true }
+      let(:registration_form) do
+        double(
```

Comment:
Use `instance_double(ClassName` so we are sure we are using the correct methods.